src/snowflake/cli/plugins/nativeapp/teardown_processor.py

```diff
@@ -141,17 +141,17 @@ def drop_package(self, auto_yes: bool):
                 needs_confirm = False
             else:
                 if needs_confirmation(needs_confirm, auto_yes):
-                    print(
+                    cc.warning(
                         f"Application package {self.package_name} was not created by SnowCLI!"
                     )
         else:
             if needs_confirmation(needs_confirm, auto_yes):
-                print(
+                cc.warning(
```

Comment:
nit: I know you didn't change it, but I'm not sure this message is clear on its own. Why is this a warning, if I'm a user?

Reply:
I can clarify the message, but the reason it is shown to the user here is because they are choosing to drop an application package with `external` distribution property, which means it was either before the CLI existed, or could be a "prod" application package that is currently associated with a listing on the marketplace.

Reply:
Yeah I get the context, I was just suggesting we clarify the message for our users.